README.md

```diff
@@ -165,7 +165,7 @@ key | default | description
 `replace` | false | replace URL without adding browser history entry
 `maxCacheLength` | 20 | maximum cache size for previous container contents
 `version` | | a string or function returning the current pjax version
-`scrollTo` | 0 | vertical position to scroll to after navigation
+`scrollTo` | 0 | vertical position to scroll to after navigation. `false` to disable scrolling.
```

Comment:
Thanks. Can it say "false to not change scroll position"? Because we aren't really "disabling" scrolling.